src/App.js

```diff
@@ -1,19 +1,56 @@
 import React from 'react';
 import './App.css';
+import TodoForm from './components/TodoForm/TodoForm';
+import TodoList from './components/TodoList/TodoList';
 
 import users from './api/users';
+import todos from './api/todos';
 
-function App() {
-  return (
-    <div className="App">
-      <h1>Add todo form</h1>
-
-      <p>
-        <span>Users: </span>
-        {users.length}
-      </p>
-    </div>
-  );
+class App extends React.Component {
+  state = {
+    todos: [...todos],
+  }
+
+  addTodo = (todo) => {
+    this.setState(prevState => (
+      prevState.todos.push(todo)
+    ));
+  }
+
+  getUserName = (todo) => {
+    const findUser = users.find(user => user.id === todo.userId);
+
+    return findUser.name;
+  }
+
+  preperTodo = () => {
```

Comment:
Краще передавай в цей метод todos
```suggestion
  prepereTodos = (todos) => {
```